src/surface/surface.scss

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+
+@mixin iui-surface($elevation: $iui-elevation-2) {
+  box-shadow: $elevation;
+  box-sizing: border-box;
+  @include themed {
+    background-color: t(iui-color-background-1);
+    color: t(iui-text-color);
+  }
+
```

Comment:
Do we need rounded borders?

Reply:
All the use cases for it currently don't require rounded corners.  Maybe we can add it later as an option if we need it?